### PR TITLE
fix the .travis.yml script

### PR DESCRIPTION
**Pull request by ares201005**

### CODE REVIEW

**Review by junghans — CHANGES_REQUESTED** [review]
What is broken with C++11? Why do you need C++98?

**Comment by ares201005** [comment]
Actually, I have another modification that turns on the c++11
compatibility, i.e., use "-std=c++11 flag". use this version?

File: .travis.yml [hunk 1]
```diff
@@ -49,7 +50,7 @@ script:
      else
        mkdir build && cd build && cmake ../cmake -DCMAKE_INSTALL_PREFIX=$HOME/latte/install -DPROGRESS=${CMAKE_WITH_PROGRESS} -DOPENMP=${CMAKE_WITH_PROGRESS} -DCMAKE_PREFIX_PATH="$HOME/bml/install;$HOME/qmd-progress/install" && make && make install && 
        if [[ ${LAMMPS_CMAKE} ]]; then
-         cd $HOME/lammps && mkdir build && cd build && cmake ../cmake -DPKG_LATTE=ON -DCMAKE_PREFIX_PATH=$HOME/latte/install && make;
+         cd $HOME/lammps && mkdir build && cd build && cmake ../cmake -DPKG_MOLECULE=ON -DPKG_LATTE=ON -DCMAKE_PREFIX_PATH=$HOME/latte/install && sed -i "s/-DLAMMPS_SMALLBIG/-DLAMMPS_SMALLBIG -DLAMMPS_CXX98/g" CMakeFiles/lmp.dir/flags.make && make;
```
[the remark this answers is not in the document]
Comment (ares201005) [reply .travis.yml]: Thanks for the hint. Another question, if I want to use c++11, how can I add "-std=c++11" to the "CMakeFiles/lmp.dir/flags.make" file using cmake?

[the remark this answers is not in the document]
Comment (junghans) [reply .travis.yml]: `-DCMAKE_CXX_STANDARD=11` or add `-std=c++11` to CXXFLAGS.

[the remark this answers is not in the document]
Comment (ares201005) [reply .travis.yml]: Thank you! I will use the "-DCMAKE_CXX_STANDARD=11" option.

**Review by cnegre — APPROVED** [review]

**Pull request merged**